### PR TITLE
Documentation: Running models

### PR DESCRIPTION
**Pull request by ljhwang**
Fixes a few formatting errors.
Still needs a correct reference for @cmillls1095 to fix :)

### CODE REVIEW

File: doc/sphinx/user/install/docker-container/running-models.md [hunk 1]
```diff
@@ -80,3 +80,12 @@ recommend the following workflow:
 
 You are all set. Repeat steps 1-4 of this process as necessary when updating
 your model parameters.
+
+:::{admonition} TODO
+:class: error
+Fix referenced link to Sectio 4.4 Debug or Optimized
```
Comment (bangerth) [inline doc/sphinx/user/install/docker-container/running-models.md:86]:
```suggestion
Fix referenced link to Section 4.4 Debug or Optimized
```

File: doc/sphinx/user/install/docker-container/running-models.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ recommend the following workflow:
     you have started the container run the aspect model inside the container.
     Note that there are two ASPECT executables
     in the work directory of the container: `aspect` and `aspect-release`. For
-    a discussion of the different versions see {ref}`4.4][], in
+    a discussion of the different versions see {ref}4.4][], in
```
Comment (bangerth) [inline doc/sphinx/user/install/docker-container/running-models.md:25]: How do we want to deal with this going forward? We can't reasonably have numbers be the labels for sections.

File: doc/sphinx/user/install/docker-container/running-models.md [hunk 3]
```diff
@@ -80,3 +80,12 @@ recommend the following workflow:
...
 your model parameters.
+
+:::{admonition} TODO
+:class: error
+Fix referenced link to Sectio 4.4 Debug or Optimized 
+:::
+
+[^footnote1]: Note that it is possible to mount a directory as writeable into the container. However, this is often associated with file
+permission conflicts between the host system and the container. Therefore, we recommend this slightly more cumbersome, but
+also more reliable workflow.
```
Comment (bangerth) [inline doc/sphinx/user/install/docker-container/running-models.md:91]: Is it not possible to have the footnote text right where the footnote symbol is? It is hard to maintain things if pieces of related text end up in different places in the document.

**Review by bangerth — CHANGES_REQUESTED** [review]

**Comment by ljhwang** [comment]
Again this has picked up unresolved conflicts so I will close as most of them have been addressed.

**Pull request closed without merging**